pkg/daemon/ceph/osd/remove.go

```diff
@@ -31,6 +32,13 @@ import (
 	"github.com/rook/rook/pkg/operator/k8sutil"
 )
 
+// CrashFindIdResult is go representation of the Ceph crash ls find command output
+type CrashFindIdResult struct {
+	ID     string `json:"ID"`
+	ENTITY string `json:"ENTITY"`
```

Comment:
nit: Use golang naming conventions even if the json does not match the same convention.
```suggestion
	Entity string `json:"ENTITY"`
```